src/ksp/withPoints_ksp.c

```diff
@@ -243,9 +230,9 @@ PGDLLEXPORT Datum _pgr_withpointsksp(PG_FUNCTION_ARGS) {
                 text_to_cstring(PG_GETARG_TEXT_P(2)),
                 NULL, NULL,
                 PG_GETARG_INT32(3),
-                PG_GETARG_BOOL(4),
+                text_to_cstring(PG_GETARG_TEXT_P(4)),
                 PG_GETARG_BOOL(5),
-                text_to_cstring(PG_GETARG_TEXT_P(6)),
+                PG_GETARG_BOOL(6),
                 PG_GETARG_BOOL(7),
                 &result_tuples,
                 &result_count);
```

Comment:
Combinations function has 8 args (you just need to update the comment): `/* (PG_NARGS() == 8) */`